frontend/src/app/main/ui/workspace/tokens/modals/export.scss

```diff
@@ -0,0 +1,125 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+//
+// Copyright (c) KALEIDOS INC
+
+@use "../../../ds/typography.scss" as t;
+@use "../../../ds/_sizes.scss" as *;
+@use "../../../ds/_borders.scss" as *;
+@import "refactor/common-refactor.scss";
+
+.modal-overlay {
+  @extend .modal-overlay-base;
+}
+
+.modal-dialog {
+  --modal-width: 30.3125rem;
```

Comment:
The value can be rounded. Better use 32rem so this is equal to other similar modals (perhaps in the design they should have used 512 instead of 485).